client/src/pages/CodeUpload.js

```diff
@@ -0,0 +1,176 @@
+import React, { useState, useContext, useEffect } from "react";
+import axios from "axios";
+import {
+  Grid,
+  Button,
+  InputLabel,
+  Select,
+  MenuItem,
+  TextField,
+  Typography,
+  makeStyles
+} from "@material-ui/core";
+import { UserContext } from "context/UserContext";
+import { TextEditor } from "components/index";
+import ErrorMessage from "components/ErrorMessage";
+import SubmitButton from "components/TextEditor/components/SubmitButton";
+
+const useStlyes = makeStyles({
+  root: {
+    padding: "5%"
+  },
```

Comment:
I think the background here looks out of place from the rest of out site. Can add `backgroundImage: "linear-gradient(to top, rgba(87,40,184,1) 25%, rgba(119,140,255,1) 100%)",` to root to get the gradient and make the site look uniform. Also probably want to have the corners rounded like in Nick's SignUpContainer component.

Reply:
Rounded the corners.

I chose that background color because I assumed from the RFP that the main site color would be this (see profile, adding credits, and reviews mockups), and only the onboarding, login, and signup pages have the purple gradient. We can discuss more on the site styling in standup.